File: csharp/atbash-cipher/AtbashTest.cs
using NUnit.Framework;

namespace Exercism
{
    [TestFixture]
    public class AtbashTest
    {
        // change Ignore to false to run test case or just remove 'Ignore = true'
        [TestCase("no", Result = "ml")]
        [TestCase("yes", Result = "bvh")]
        [TestCase("OMG", Result = "lnt")]
        [TestCase("mindblowingly", Result = "nrmwy oldrm tob")]
        [TestCase("Testing, 1 2 3, testing.", Result = "gvhgr mt123 gvhgr mt")]
        [TestCase("Truth is fiction.", Result = "gifgs rhurx grlm")]
        [TestCase("The quick brown fox jumps over the lazy dog.", Result = "gsvjf rxpyi ldmul cqfnk hlevi gsvoz abwlt")]
        public string Encodes_words_using_atbash_cipher(string words)
        {
            return Atbash.Encode(words);
        }
    }
}

File: v1/fsharp/atbash-cipher/AtbashTest.fs
module AtbashTests

open NUnit.Framework
open Atbash

[<TestCase("no", ExpectedResult = "ml")>]
[<TestCase("yes", ExpectedResult = "bvh")>]
[<TestCase("OMG", ExpectedResult = "lnt")>]
[<TestCase("mindblowingly", ExpectedResult = "nrmwy oldrm tob")>]
[<TestCase("Testing, 1 2 3, testing.", ExpectedResult = "gvhgr mt123 gvhgr mt")>]
[<TestCase("Truth is fiction.", ExpectedResult = "gifgs rhurx grlm")>]
[<TestCase("The quick brown fox jumps over the lazy dog.", ExpectedResult = "gsvjf rxpyi ldmul cqfnk hlevi gsvoz abwlt")>]
let ``Encodes words using atbash cipher`` (words: string) =        
    encode words